src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala

```diff
@@ -50,22 +50,6 @@ class FailureHandlerSpec extends LambdaSpec {
 
   }
 
-  "FailureHandler lambda" should "still return a failed JsonWrapper and an Unknown failure reason if it receives the old schema" in {
```

Comment:
We will never be receiving the old schema any more so I've removed code and tests that refer to it